### PR TITLE
global: migrate CI to gh-actions

### LINKED ISSUE #123: global: migration to github-actions from travis

### PR DESCRIPTION
**Pull request by ParthS007**
closes #123

### CODE REVIEW

File: .github/workflows/js-tests.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```
Comment (diegodelemos) [inline .github/workflows/js-tests.yml:34]: Might the error in the build be related to the fact that before this package was using node 6 and 7? see [here](https://github.com/inveniosoftware/invenio-search-js/blob/49a5000cdc9ca7b33d1b0d27f02cf75026ab23a9/.travis.yml#L17-L18).

Comment (ParthS007) [reply .github/workflows/js-tests.yml:34]: Okay, let me check with the lower version 👍

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
```
Comment (diegodelemos) [inline .github/workflows/npm-publish.yml:27]: Wouldn't it be dangerous to use a different node version for the release? It used to use `6.9.1` see https://github.com/inveniosoftware/invenio-search-js/pull/124/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L49.

Comment (ParthS007) [reply .github/workflows/npm-publish.yml]: Done 👍

**Review by diegodelemos — APPROVED** [review]

**Pull request merged**